doc/api/http.md

```diff
@@ -747,6 +747,11 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: in most operating systems, listening to `::`(_[unspecified IPv6
```

Comment:
@sam-github thanks, the markup does work( see https://github.com/sjitech/node/blob/doc-for-listen-all/doc/api/http.md#serverlistenport-hostname-backlog-callback)

I do want to change for both v4 and v6, but i am not sure it's OK to do so. 

@gibfahn thanks!